### PR TITLE
IBM and Red Hat updates.

### PR DESCRIPTION
**Pull request by ipanova**

### CODE REVIEW

**Review by bmbouter — APPROVED** [review]
This looks great, thank you @ipanova :+1:

File: _posts/2019-07-09-redhat-and-ibm.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+author: Ina Panova
+tags:
+  - community 
+---
+
+Pulp community,
+
+Today marks a new day in the 26-year history of Red Hat. [IBM has finalized its acquisition of
+Red Hat](https://www.redhat.com/en/about/press-releases/
+ibm-closes-landmark-acquisition-red-hat-34-billion-defines-open-hybrid-cloud-future), which will
```
Comment (daviddavis) [inline _posts/2019-07-09-redhat-and-ibm.md]: I think the linebreak here needs to be removed. It's not displaying the link correctly for me.

File: _posts/2019-07-09-redhat-and-ibm.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Our mission, governance, and objectives remain the same. We will continue to execute the existing
+project roadmap. Red Hat associates will continue to contribute to the upstream in the same ways
+they have been. And, as always, we will continue to help upstream projects be successful and
+contribute to welcoming new members and maintaining the project.
+
+We will do this together, with the community, as we always have.
+
+ 
+If you have questions or would like to learn more about today’s news, We encourage you to review
```
Comment (daviddavis) [inline _posts/2019-07-09-redhat-and-ibm.md]: s/We/we/

File: _posts/2019-07-09-redhat-and-ibm.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+the list of materials below. Red Hat CTO Chris Wright will host an online Q&A session in the coming
+days where you can ask questions you may have about what the acquisition means for Red Hat and our
+involvement in open source communities. Details will be announced on the
+[Red Hat blog](https://www.redhat.com/en/blog).
+
+ * Chris Wright blog - [Red Hat and IBM: Accelerating the adoption of open source](https://
+   www.redhat.com/en/blog/red-hat-and-ibm-accelerating-adoption-open-source)
```
Comment (daviddavis) [inline _posts/2019-07-09-redhat-and-ibm.md]: Link is broken here too.

**Review by daviddavis — APPROVED** [review]

**Pull request merged**